test/exe/render_test.rb

```diff
@@ -17,6 +17,33 @@ def test_render_parses_paths
     krane_render!("--filenames #{paths}")
   end
 
+  def test_render_parses_std_in
+    Dir.mktmpdir do |tmp_path|
+      file_path = "/dev/null"
+      $stdin.expects("read").returns("")
+      Dir.expects(:mktmpdir).with("krane").yields(tmp_path)
+      install_krane_render_expectations(template_paths: [file_path, tmp_path])
+      krane_render!("--filenames #{file_path} --std-in")
+    end
+  end
+
+  def test_render_parses_std_in_without_filenames
+    Dir.mktmpdir do |tmp_path|
+      $stdin.expects("read").returns("")
+      Dir.expects(:mktmpdir).with("krane").yields(tmp_path).once
+      install_krane_render_expectations(template_paths: [tmp_path])
+      krane_render!("--std-in")
+    end
+  end
+
+  def test_render_fails_without_filename_and_std_in
```

Comment:
There is something flakey or not working about this test `rt test/exe/render_test.rb --seed 58960` reproduces it. 

Somehow render task is getting called with these options `{"filenames"=>["-"], "std-in"=>false, "current-sha"=>"1"}`